starter-project-web-react/README.md

```diff
@@ -1,5 +1,13 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
+## starter_next_project
+
+1. Next, TypeScript, Redux, MUI, and Jest will be used in this project
+2. Add a page to the pages folder with your task on it (eg. auth.tsx)
+3. Add your page's component to the components folder with your name (eg . JohnLoginButton.tsx)
```

Comment:
i think we should forget the part about using their names
there is no need for it 
and we can just check the git history if needed